src/sentry/replaystore/django/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from sentry.db.models import BaseModel, sane_repr
+
+
+class Replay(BaseModel):  # type: ignore
+    __include_in_export__ = False
+
+    id = models.CharField(max_length=80, primary_key=True)
```

Comment:
Since you're using `startswith` on this column, you'll need to add an index like 
`CREATE INDEX "replaystore_replay_id_like_idx" ON "replaystore_replay" ("id" varchar_pattern_ops);`

That said, I think you might actually be better off defining this as multiple columns, like:
```
key = models.CharField(max_length=80)
type = models.CharField(max_length=80)
timestamp = models...
```

and add a unique index on `(key, type, timestamp)`. That would mean passing these values individually to each storage backend instead of key, but I think that makes sense, since currently we leak the implementation details of how we store this in bigtable to other implementations

Reply:
yeah, i went back and forth on that. went ahead and added them as separate fields with an index. in case this ends up being used by people other than just dev testing, it is a better choice to do it this way.